docs/bookdown/developer-guide/01-prereqs.Rmd

```diff
@@ -2,7 +2,7 @@
 
 ## Platform Support {#plat-support}
 
-Plugins developed with the SDK require the RStudio Launcher to work. The SDK is currently compatible with the version of the RStudio Launcher that is shipped with RStudio Server Pro v1.4. As such, the SDK will be supported on all of the platforms supported by RStudio Server Pro v1.4, which can be found on the [Requirements for RStudio Server Pro page](https://docs.rstudio.com/rsp/requirements/).
+Plugins developed with the SDK require the RStudio Launcher to work. The SDK is currently compatible with the version of the RStudio Launcher that is shipped with RStudio Workbench v1.4.1. As such, the SDK will be supported on all of the platforms supported by RStudio Workbench v1.4.1, which can be found on the [Requirements for RStudio Workbench page](https://docs.rstudio.com/rsp/requirements/).
```

Comment:
Since the SDK will still work with RSP 1.4 I don't want to be so specific about the version. RStudio Workbench v1.4 will still be a valid version. Can we leave the patch number out?

Reply:
I thought about leaving it to begin with. I'll make that change.